lambda-src/lambda_function.py

```diff
@@ -0,0 +1,23 @@
+import json
```

Comment:
what is the goal of the function? Maybe give the folder or file a better name

Reply:
I only did need it for the test print in line 10 you commented. It's a really early version of the lambda function.